example.go

```diff
@@ -79,7 +79,7 @@ func getEnvVarOrExit(varName string) string {
 }
 
 func createClients(subscriptionID string, authorizer *autorest.BearerAuthorizer) {
-	sampleUA := fmt.Sprintf("Azure-Samples/storage-go-resource-provider-getting-started/%s", utils.GetCommit())
+	sampleUA := fmt.Sprintf("sample/0001/%s", utils.GetCommit())
```

Comment:
I agree with @jhendrixMSFT, there's some real value to having a readable user agent string, and the difference of payload is pretty much negligible. If we'd like unique identifiers for the samples, I would suggest we do so in an additive fashion.